doc/source/quickstart/quickstart-hologres-with-tableau.md

```diff
@@ -0,0 +1,856 @@
+
+If under Linux, go with the following script and download the data；
+One thing to remember, choose one specific directory before you start （data would be downloaded under the current directory by default).
+
+Note: in 1987, only 10,11,12 (3 months) data is available for download.
+
+
+```
+for s in `seq 1987 2018`
+do
+for m in `seq 1 12`
+do
+wget https://transtats.bts.gov/PREZIP/On_Time_Reporting_Carrier_On_Time_Performance_1987_present_${s}_${m}.zip
+done
+done
+```
+
+
+![image](../images/Quickstart/quickstart-Hologres/datadownload1.gif)
+
+![image](../images/Quickstart/quickstart-Hologres/datadownload2.gif)
+
+Data is stored in CSV (with ',' delimiter) format. Total uncompressed CSV files size is about 79852548 Bytes (about 77GB).
+
+Note: the following 14 files contain invalid byte sequence for encoding, and thus fails to upload to Hologres.
+
+| NO | FILES |
+|---|---|
+| 1 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_1.csv |
+| 2 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_2.csv |
+| 3 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_3.csv |
+| 4 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_4.csv |
+| 5 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_5.csv |
+| 6 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_6.csv |
+| 7 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_7.csv |
+| 8 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_8.csv |
+| 9 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_9.csv |
+| 10 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_10.csv |
+| 11 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_11.csv |
+| 12 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_12.csv |
+| 13 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2002_1.csv |
+| 14 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2002_2.csv |
+
+
+Let's delete these 14 files via the following commands:
+
+```
+# remove the files in 2001 .12 files in total  
+rm -f On_Time_Reporting_Carrier_On_Time_Performance_\(1987_present\)_2001*.csv
+
+# remove the files in 2002_1 and 2002_2 , 2 files in total  
+rm -f On_Time_Reporting_Carrier_On_Time_Performance_\(1987_present\)_2002_1.csv
+rm -f On_Time_Reporting_Carrier_On_Time_Performance_\(1987_present\)_2002_2.csv
+```
+
+3.Unzip the Dataset 
+
+Once download the data, unzip it with the following script:
+
+```
+find . -name '*.zip'-exec unzip {} \；
+```
+
+The following warning might come, choose "N":
+```
+#replace readme.html? [y]es, [n]o, [A]ll, [N]one, [r]ename: 
+```
+
+## Create Table in Hologres 
+
+Let's connect to Hologres via psql with the following command.
+
+```
+PGPASSWORD=<password> psql -U <username> -h <ip> -p <port> -d <database>
+```
+
+Now you have landed Hologres successfully(there shoud be no database by default).
```

Comment:
if "there shoud be no database by default)", what's the database you connect to in "PGPASSWORD=<password> psql -U <username> -h <ip> -p <port> -d <database>"?